pkg/universe.dagger.io/x/carvel.dev/kapp/kapp.cue

```diff
@@ -0,0 +1,223 @@
+package kapp
+
+import (
+	"dagger.io/dagger"
+	"universe.dagger.io/docker"
+)
+
+//This is an experimental feature.
+//Currently works with 'kind' cluster
+//Can be imported as universe.dagger.io/x/carvel.dev/kapp
+
+_#entryPoint: {
+	deployScript: #"""
+		    if [ -d /source ] || [ -f /source ]; then
+		        kapp deploy -n "$KAPP_NAMESPACE" -a "$APP_NAME" -f /source -y
+		        exit $?
+		    fi
+		    if [ -n "$DEPLOYMENT_URL" ]; then
+		        kapp deploy -n "$KAPP_NAMESPACE" -a "$APP_NAME" -f "$DEPLOYMENT_URL" -y
+		        exit $?
+		    fi
```

Comment:
instead of doing this conditional logic in bash, why dont we do it in cue when constructing command. (goes with the below comment of how we can get rid of using a shell)